config/src/config/dag_consensus_config.rs

```diff
@@ -0,0 +1,156 @@
+use super::{config_sanitizer::ConfigSanitizer, node_config_loader::NodeType, Error, NodeConfig};
+use aptos_types::chain_id::ChainId;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct DagNodePayloadConfig {
+    pub max_sending_txns: u64,
+    pub max_sending_size_bytes: u64,
+    pub max_receiving_txns: u64,
+    pub max_receiving_size_bytes: u64,
+
+    pub payload_pull_max_poll_time_ms: u64,
+}
+
+impl Default for DagNodePayloadConfig {
+    fn default() -> Self {
+        Self {
+            max_sending_txns: 1000,
```

Comment:
I think we probably need to have a per round limit and then divide by the number of validators in each epoch